src/components/pages/Offerers/OfferersContainer.jsx

```diff
@@ -10,6 +11,22 @@ import { selectOfferers } from '../../../selectors/data/offerersSelectors'
 
 import { OFFERERS_API_PATH } from '../../../config/apiPaths'
 
+export const createApiPath = searchKeyWords => {
+  let apiPath = OFFERERS_API_PATH
+  if (Array.isArray(searchKeyWords)) {
+    searchKeyWords = searchKeyWords.join(' ')
+  }
+  const isKeywordValidParam =
+    searchKeyWords !== undefined && searchKeyWords !== null && searchKeyWords !== ''
+
+  if (isKeywordValidParam) {
```

Comment:
pour quoi pas juste
```suggestion
  if (searchKeyWords) {
```
undefined, null et '' sont false